Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.cpp

```diff
@@ -1249,7 +1368,21 @@ std::optional<unsigned> ResourceLoadStatisticsStore::domainID(const RegistrableD
 {
     ASSERT(!RunLoop::isMain());
 
-    auto scopedStatement = this->scopedStatement(m_domainIDFromStringStatement, domainIDFromStringQuery, "domainID"_s);
+    return domainIDWithTable(domain, m_domainIDFromStringStatement, domainIDFromStringQuery);
+}
+
+std::optional<unsigned> ResourceLoadStatisticsStore::loginStatusDomainID(const RegistrableDomain& domain) const
+{
+    ASSERT(!RunLoop::isMain());
+
+    return domainIDWithTable(domain, m_loginstatusDomainIDFromStringStatement, loginStatusdomainIDFromStringQuery);
```

Comment:
Status should have a capital S, and Domain should have a capital D in both variable names.